commands/apply_referral_code_test.go

```diff
@@ -10,31 +10,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestJoinTeam(t *testing.T) {
+func TestApplyReferralCode(t *testing.T) {
 	t.Run("Joining team succeeds", testJoiningTeamSucceeds)
-	t.Run("Joining team with team ID fails", testJoinTeamWithoutTeamIDFails)
+	t.Run("Joining team with team ID fails", testApplyReferralCodeWithoutTeamIDFails)
```

Comment:
How can this be true if there's no such thing as a team ID anymore?

Reply:
it's just naming of a test, that's fine. teams still exists but in a different form